client/my-sites/site-settings/seo-settings/main.jsx

```diff
@@ -12,34 +11,10 @@ import notices from 'notices';
 import QuerySitePurchases from 'components/data/query-site-purchases';
 import QuerySiteSettings from 'components/data/query-site-settings';
 import { getSitePurchases, hasLoadedSitePurchasesFromServer, getPurchasesError } from 'state/purchases/selectors';
-import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSelectedSiteId, getSelectedSite } from 'state/ui/selectors';
 import SeoForm from './form';
```

Comment:
In the `SeoForm` we'll also have to port the settings retrieval and saving to Redux:

* `site.saveSettings`
* `site.fetchSettings`

Although this seems pretty straightforward, at first glance I notice that `/me/sites` does not return consistent `options.advanced_seo_title_formats` - it returns an empty array, while the settings endpoint returns the correct values. We'll probably have to fix this on an API level. Other SEO settings might be failing the same way, too - we should check them all.